common.tf

```diff
@@ -7,8 +7,8 @@
 locals {
   instance_alias         = "${ var.instance_name == "" ? "apiary" : format("apiary-%s",var.instance_name) }"
   enable_route53_records = "${ var.apiary_domain_name == "" ? "0" : "1" }"
-  apiary_managed_schemas = "${ split(",",replace(join(",",var.apiary_managed_schemas),"_","-")) }"
-  apiary_data_buckets    = "${ formatlist("%s-%s-%s-%s",local.instance_alias,data.aws_caller_identity.current.account_id,var.aws_region,local.apiary_managed_schemas) }"
+  apiary_apiary_managed_schema_names =  ["${data.template_file.schema_names.*.rendered}"]
```

Comment:
Please format this as the other files are formatted. You can run `terraform fmt` on your project to format all the files in the consistent manner.

Reply:
sure, thanks

Reply:
why `apiary_apiary`?
Also, how does this achieve the same functionality of replacing `_` with `-`?

Reply:
var.apiary_managed_schemas and local.apiary_managed_schemas were being used for different purpose. local.apiary_managed_schemas for S3 bucket creation.

so you need 2 local variables, probably apiary_managed_schemas_original, apiary_managed_schemas_replaced 

and you can use following for apiary_managed_schemas_replaced  tempalte.
I think you need to use replace(lookup(var.apiary_managed_schemas[count.index], "schema_name"),"_","-") in the template

Reply:
Okay sure, I will make the above change